software/controller/lib/hal/spi_chain.h

```diff
@@ -72,10 +71,21 @@ class DaisyChain : public Channel, public RxListener {
   void on_rx_error(RxError) override{};
```

Comment:
Nothing happens on rx error or character match?
Maybe some comment on why this interface is inherited but not used?